samples/speech/index.html

```diff
@@ -71,19 +73,23 @@
             
 
             // Note that chrome automatically blocks speech if the html file is loaded from disk. You can run a server locally 
-            // or launch chrome with the following option:
+            // or launch chrome (close all the existing chrome browsers) with the following option:
             // chrome.exe --allow-file-access-from-files <sampleHtmlFile>
-            var speechOptions = {
-                speechRecognizer: new BotChat.SpeechRecognition.BrowserSpeechRecognizer(),
-                speechSynthesizer: new BotChat.SpeechSynthesis.BrowserSpeechSynthesizer()
-            }
+            // var speechOptions = {
```

Comment:
We provide browser speech for free. And the Cognitive Service is provided as a separate library.